vllm/worker/worker_base.py

```diff
@@ -519,6 +520,12 @@ def __init__(
                 from vllm.utils import init_cached_hf_modules
                 init_cached_hf_modules()
 
+    def __del__(self):
+        self.shutdown()
+
+    def shutdown(self) -> None:
+        ensure_kv_transfer_shutdown()
```

Comment:
any chance this could leave within a the mixin https://github.com/vllm-project/vllm/blob/main/vllm/v1/worker/kv_connector_model_runner_mixin.py?

Reply:
good idea.